### ISSUE TITLE
Feature Request: log to custom URL

### ISSUE THREAD

**Issue by blackno666** [BasicAirData/GPSLogger#203]
It would be great if you could implement functionality to log to a custom URL - as do other trackers, like ulogger, OsmAnd, GpsLogger, Owntracks, Tracccar

**Comment by mtangoo** [comment]
I came across this app looking for an app with similar functionality but that logs to the server.
This came closed but this feature missing was deal breaker. I guess I will have to write myself!

**Comment by JLJu** [comment]
" I guess I will have to write myself!!
To code and share on Github is always good :-)

**Comment by mtangoo** [comment]
Yes, but it will be simple "Throw data to URL". I want just simple tests.
Won't be worthy a github repo. Not as serious as this great project ;)
